src/rpc/cli.cpp

```diff
@@ -18,10 +18,10 @@
 
 namespace fc { namespace rpc {
 
-static std::string& cli_regex_secret()
+static boost::regex& cli_regex_secret()
 {
-   static std::string* regex_secret = new std::string();
-   return *regex_secret;
+   static boost::regex* regex_expr = new boost::regex();
```

Comment:
You still use ``new`` operator.

Reply:
`boost::regex` must be initialized, it cannot be null

Reply:
without `new` operator we will have `SEGFAULT`

```
./programs/cli_wallet/cli_wallet --server-rpc-endpoint=wss://node.bitshares.eu
Logging RPC to file: logs/rpc/rpc.log
687810ms th_a       main.cpp:147                  main                 ] key_to_wif( committee_private_key ): 5KCBDTcyDqzsqehcb52tW5nU6pXife6V2rX9Yf7c3saYSzbDZ5W
687811ms th_a       main.cpp:151                  main                 ] nathan_pub_key: BTS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV
687811ms th_a       main.cpp:152                  main                 ] key_to_wif( nathan_private_key ): 5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3
687814ms th_a       main.cpp:199                  main                 ] wdata.ws_server: wss://node.bitshares.eu
688461ms th_a       main.cpp:204                  main                 ] wdata.ws_user:  wdata.ws_password:
Segmentation fault (core dumped)
```

Reply:
You have to remove the ``*`` as well of course.

Reply:
ok sure, I will not use `pointer`, will fix it now. Thanks !